2018q4/bcm58712.md

```diff
@@ -0,0 +1,11 @@
+## Broadcom ARM64 SoC support ##
+
+Contact: Michal Stanek, <mst@semihalf.com>, Marcin Wojtas, <mw@semihalf.com>
+
+Semihalf has recently started work on FreeBSD support for BCM5871X SoC series. These are quad-core 64-bit ARMv8 Cortex-A57 communication processors targeted for
+networking applications such as 10G routers, gateways, control plane processing and NAS. Initial support will include iProc PCIe, internal BNXT Ethernet
+controller, OTP (One Time Programmable memory)
+and crypto engine acceleration for IPsec offloading.
+
```

Comment:
Are you able to make some comment about planned timelines? I understand it may be sensitive or there may be details you cannot share, but if you're able to add a comment like "this work is expected to be ready for FreeBSD-HEAD before Q3 2019" or such that would benefit the report IMO.

Reply:
Done.